.github/workflows/arm64.yml

```diff
@@ -96,3 +102,20 @@ jobs:
           docker volume rm -f $(docker volume ls -q) || true
           docker rmi -f $(docker images -aq) || true
           docker system prune -a -f
+  create_jira_issue:
+    name: "Create JIRA issue"
```

Comment:
I would rename this to `jira_integration` and `"JIRA Integration"` respectively, as it's not necessarily creating a new JIRA issue. It might close a pre-existing one or do nothing at all as well.

Reply:
Done.